explicit.js

```diff
@@ -2,7 +2,7 @@ function P(target){
   for(var i = 1; i < arguments.length; i++)
     for(var key in arguments[i])
       if(arguments[i].hasOwnProperty(key))
-        arguments[i][key] === D
+        (arguments[i][key] === D || arguments[i][key] === O)
```

Comment:
The second expression isn't desirable, the intent is that `O` is always used as a function (and returns `D` in this case).

Reply:
[eg](https://flems.io/#0=N4IgZglgNgpgziAXAbVAOwIYFsZJABwwBcBjACwjQGsB7AOgCsEAaEEmtImTvMAVzQkiEDgAIACgAoiGAE4BzGEQCUwADppRosDVmSAbnNERRAXlEBGANzHRAHlFz5fHJzh1YaeUTI2IAan9lDS0tHT1DWVEqGABPY00nF24iOGQIAF1gzVCtCDBJJNdU9Iy6Mgw4AHkAdzRxWRp8GFkiWMkY2OVs3NzChWS3UuROjLNTcwARHt6tAH5RABMYWC5RGQUlEbiMkNnRRHWnLdGzPf3HAeK0zO3Ysco4GUEYGjBRAGVz-YWilJuMncyhh8PgoO0NooiECZhdDn8hrdRho9rIlHxZJpIUoNABfFFofiCYRiD6SEhQGhwDEwVR7fKSACE0gocASTwwLzen263zRRAxmjQMBqn3JlOpaOyex8EHcILB8XMRKEIjQkmWkDQEBJaDpOS0-MFogpVJpGpgWp1apm+LQdo0Kt1EjJ2KIzAS+D4KnUOSNmNEwtFZO+CNSHm43jI43MACZvgsnWqLVbdfrZv7NFI3R7NZRrRwPZQvSpvnbcockxwU-m076M+iA1I89rdR63bDROXRNkHYSBKqxJNJKo+1XNFVJAAPdOiTOXZzXCNeHwJyz2BeDcOeKPfeYSOgK8GSNB8KBQD1huCdrSHNrNblTmOiADk45fe9CCyDYpnn9vLrTp2hyTHiBJELI8T1qIWA0IsZ4wHQMBTvguipGYojAOIhziB6HyHB8HriPhLoepMIEelUhxVHauImsQ5CSLSmG4iArBwCsMCDmgCCICAAAsACciAWAAzCAuLMOg2C4HxjAsGwHBcDwfH4JIeyTt80G5BgByWMwn4AEZ6bGBkGl2Zm5Fpn66dRhRmAAfI4ogALSWMolm9MZdkeWWnnZMobEgBxsDcbxIAWAADIgABsEkZLiQA)

Reply:
Then there is a discrepancy with the README, which has:

```
bish: O
```

also, `O` from `overloaded.js` doesn't work with `O()` to delete.

Reply:
You're right - sorry!